modules/csharp/pages/gs-build.adoc

```diff
@@ -30,6 +30,31 @@ The document was successfully persisted to the database.
 
 See also -- {sgw-install--xref}
 
+=== Building With Vector Search
+
+To build your Couchbase Lite application with Vector Search, you must add the following to your existing Couchbase Lite application:
+
+. Add the Vector Search import statement
++
+[source, csharp]
+----
+
+using Couchbase.Lite.Extensions;
+
+----
++
+. Load the extension.
++
+[source, csharp]
+----
+// For Android applications
+Couchbase.Lite.Support.Droid.Activate(Context);
+Extension.Load(new VectorSearchExtension(Context));
```

Comment:
Warning:  These will not get checked by the compiler in the way that snippets will.

Reply:
Done!